robottelo.properties.sample

```diff
@@ -1,106 +1,137 @@
 # Make a copy of this file named robottelo.properties
+[server]
+# Server hostname
+hostname=
 
-[main]
+# Path to private ssh key to be used when connecting via SSH.
+ssh_key=
 
-# A hostname is required. scheme (default: https) and port are optional.
+# HTTP scheme when building the server URL
 # Suggested values for "scheme" are "http" and "https".
-server.hostname=
-#server.scheme=https
-#server.port=
+# scheme=https
 
-server.ssh.key_private=/home/whoami/.ssh/id_hudson_dsa
-server.ssh.username=root
+# Server port
+# port=443
 
-# For LDAP Authentication.
-ldap.hostname=
-ldap.username=
-ldap.passwd=
-ldap.basedn=
-ldap.grpbasedn=
+# SSH username
```

Comment:
One thing is ssh connection to the machine, the other is credentials to access the system via HTTP using the API or UI endpoints. They should be different credentials because admin user from system don't need to be a user on the linux system.